src/lib.rs

```diff
@@ -48,10 +57,9 @@ pub use seeded_state::FxSeededState;
 #[cfg(feature = "std")]
 pub use seeded_state::{FxHashMapSeed, FxHashSetSeed};
 
-/// A speedy hash algorithm for use within rustc. The hashmap in liballoc
-/// by default uses SipHash which isn't quite as speedy as we want. In the
-/// compiler we're not really worried about DOS attempts, so we use a fast
-/// non-cryptographic hash.
+/// A speedy hash algorithm for use within rustc.
+///
+/// The hashmap in liballoc by default uses SipHash which isn't quite as speedy as we want. In the compiler we're not really worried about DOS attempts, so we use a fast non-cryptographic hash.
```

Comment:
this line is now very long

```suggestion
/// The hashmap in liballoc by default uses SipHash which isn't quite as speedy as we want.
/// In the compiler we're not really worried about DOS attempts, so we use a fast non-cryptographic hash.
```